tests/phpunit/unit/Session/Handler/Factory/MemcachedFactoryTest.php

```diff
@@ -0,0 +1,636 @@
+<?php
+
+namespace Bolt\Tests\Session\Handler\Factory;
+
+use Bolt\Session\Handler\Factory\MemcachedFactory;
+use Bolt\Session\OptionsBag;
+use Bolt\Tests\Session\Handler\Factory\Mock\MockMemcached;
+use Memcached;
+use PHPUnit_Framework_TestCase as TestCase;
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+/**
+ * @requires extension memcached 2.0.0
```

Comment:
For some reason it is the "`2.0.0`" that is triggering the same errors for me, as it is Travis.

Removing "`2.0.0`" causes the tests to correctly skip locally.